packages/ts/react-crud/src/autocrud.tsx

```diff
@@ -146,6 +146,7 @@ export function AutoCrud<TModel extends AbstractModel>({
           setItem(activeItem ?? undefined);
         }}
         ref={autoGridRef}
+        aria-controls={`${id ?? 'auto'}-form`}
```

Comment:
In case the there are more than one AutoCrud in a view, and the `id` prop is `undefined/null`, the AutoForm's `id` and the `aria-controls` of AutoGrid would clash, right?

Reply:
Now id is generated and will be unique everytime